notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -5496,6 +5496,44 @@ mother and daughter [Isabel interacted with her mother, Jennifer.]
 father and daughter [It was strongly suggested that Isabel's father brought her down to his private study late at night to molest her.]
 
 
+movie: The Wild Blue Yonder (2005)
+==================================
+
+:: Title
+The Wild Blue Yonder
+
+:: Description
+The Wild Blue Yonder is a 2005 science fiction fantasy film by German director
+Werner Herzog.
+
+Synopsis: A team of astronauts are sent on a mission to discover a new home
+for the human race. Most of the film consists of recontextualized documentary
+footage which is overlaid with the fictional narration of a man purporting to
+be an alien from the outer reaches of the Andromeda galaxy.
+
+:: Date
+2005-09-05
+
+:: References
+https://en.wikipedia.org/wiki/The_Wild_Blue_Yonder
+
+:: Ratings
+4 <paul>
+
+:: Major Themes
+extraterrestrial being [An unnamed extraterrestrial, who was indistinguishable from a human being in form, recounted how he traveled to Earth from the outer reaches of Andromeda with other members of his race to colonize the planet, but without success.]
+alien brought pandemic [A team of astronauts were sent on a mission to find a new home for humanity after scientists mistakenly reasoned that a deadly alien disease had started to proliferate around the Earth.]
+space travel [A team of astronauts astronauts were sent on a mission initially to the outer reaches of solar system in search of habitable planet, although they eventually seemed to have traveled much deeper into outer space.]
+space colonization [The extraterrestrial narrator described how he and some of of people traveled to the outer reaches of the Andromeda galaxy to Earth to set up a colony, but without success. Humanity began a search for a new habitable planet to colonize after scientists mistakenly concluded that a deadly alien disease had started to proliferate around the Earth.]
+ocean planet [A team of astronauts discovered a glaciated ocean planet  and explored its depths. The extraterrestrial narrator's home world was an ocean planet located in the outer reaches of the Andromeda galaxy.]
```

Comment:
Double spaces detected

Reply:
What can I say, Herzog memorized me with the re-contextualized archival footage of a 1989 space shuttle Atlantis mission.